### ISSUE TITLE
Not possible to install preview version of ExchangeOnlineManagement following the documentation

### ISSUE THREAD

**Issue by ricmik** [MicrosoftDocs/office-docs-powershell#6773]
Installing a preview module following this documentation does not work.

1. -RequiredVersion does not allow "-Preview" to be a part of the version number
```
PS > Install-Module -Name ExchangeOnlineManagement -RequiredVersion 2.0.4-Preview2 -AllowPrerelease

Install-Module: Cannot process argument transformation on parameter 'RequiredVersion'. Cannot convert value "2.0.4-Preview2" to type "System.Version". Error: "Input string was not in a correct format."
```

2. -AllowPrerelase is not a valid switch
```
PS > Install-Module -Name ExchangeOnlineManagement -RequiredVersion 2.0.4 -AllowPrerelease

Install-Module: A parameter cannot be found that matches parameter name 'AllowPrerelease'.
```

```
PS > $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.0.3
PSEdition                      Core
GitCommitId                    7.0.3
OS                             Microsoft Windows 10.0.19041
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: de2836a0-0af3-c147-2d4a-78220ac4e0f1
* Version Independent ID: 2cde9735-6281-bb9f-bf07-b30b2f6c7f37
* Content: [About the Exchange Online PowerShell V2 module](https://docs.microsoft.com/en-us/powershell/exchange/exchange-online-powershell-v2?view=exchange-ps#install-and-maintain-the-exo-v2-module)
* Content Source: [exchange/docs-conceptual/exchange-online-powershell-v2.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/docs-conceptual/exchange-online-powershell-v2.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@ricmik, something else is going on. The _AllowPrerelease_ switch is definitely available in PowerShell 5.1 and 7.x, at least if the help for [Install-Module](https://docs.microsoft.com/en-us/powershell/module/powershellget/install-module) is to be believed.

**Comment by ricmik** [comment]
Both issues were solved by updating the PowershellGet module.
